lib/uring/uring.ml

```diff
@@ -148,6 +148,7 @@ module Sketch = struct
     if alloc_len > avail t then begin
       let new_buffer = create_buffer ((length t) + alloc_len) in
       t.old_buffers <- t.buffer :: t.old_buffers;
+      (* Printf.printf "len t.old_buffers = %d\n%!" (List.length t.old_buffers); *)
```

Comment:
debug leftover

Reply:
OK, I've removed this and added a new `Uring.Stats` module, which allows you to get some stats about how the ring and sketch buffer are behaving.